ovos_plugin_manager/templates/agent_tools.py

```
@@ -0,0 +1,208 @@
+            toolbox_id: A unique identifier for this ToolBox instance (usually the entrypoint name, e.g., 'web_search_tools').
+            bus: The OVOS Messagebus client instance. If provided, `bind()` is called automatically.
+        """
+        self.toolbox_id: str = toolbox_id  # Unique ID for the toolbox
+        self.bus: Optional[Union[MessageBusClient, FakeBus]] = None
+
+        # Internal cache for discovered tools, mapped by name
+        self.tools: Dict[str, AgentTool] = {}
+        try:
+            self.discover_tools() # try to find tools immediately
+        except Exception as e:
+            pass  # will be lazy loaded or throw error on first usage
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)